torchtune/datasets/_slimorca.py

```diff
@@ -4,16 +4,20 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from torchtune.data import Llama2ChatFormat, sharegpt_to_llama2_messages
+from typing import Optional
+
+from torchtune.data import ChatFormat, Llama2ChatFormat, sharegpt_to_llama2_messages
 
 from torchtune.datasets._chat import ChatDataset
 
 from torchtune.modules.tokenizers import Tokenizer
 
 
 def slimorca_dataset(
+    *,
     tokenizer: Tokenizer,
     source: str = "Open-Orca/SlimOrca-Dedup",
+    chat_format: Optional[ChatFormat] = Llama2ChatFormat,
```

Comment:
should this still be a required parameter? this is not easy to configure from the config (without a nested component)

Reply:
Yeah it's not perfect but see my test command, now I can at least disable to run Llama3 without the chat format from CLI alone